SecretOfManaClone.py

```diff
@@ -1,19 +1,123 @@
 # SecretOfManaClone.py
 import pygame
+import time
+from itertools import cycle
 
 KEY_UP = 273
 KEY_DOWN = 274
 KEY_LEFT = 276
 KEY_RIGHT = 275
 
 
-def attack_animation(direction):
-    pass
+def load_attack_animation(direction):
+    images = []
+    if direction == "down":
+        images.append(pygame.image.load("./art/dk_down_attack1.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_attack2.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_attack3.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_attack4.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_attack5.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_attack6.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_attack7.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_attack8.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_attack9.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_attack10.png").convert_alpha())
```

Comment:
That's a lot of copy pasted lines, could we use a loop for this?

Reply:
Thanks, I was able to get rid of quite a few lines of code with these suggestions.